packages/components/src/remix-hook-form/data-table-router-toolbar.tsx

```
@@ -1,149 +1,88 @@
 export interface DataTableRouterToolbarProps<TData> {
   table: Table<TData>;
-  filterableColumns?: DataTableFilterableColumn<TData>[];
-  searchableColumns?: DataTableSearchableColumn<TData>[];
-  setUrlState: (state: Partial<DataTableRouterState>) => void;
-  defaultStateValues: DataTableRouterState;
+  search?: string;
+  onSearchChange: (newSearch: string) => void;
+  onResetFiltersAndSearch: () => void;
+  hasActiveFiltersOrSearch: boolean;
+
+  dtfColumns: any[];
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)